recipe/meta.yaml

```
@@ -24,7 +23,7 @@ requirements:
   run:
     - python
     - numpy >1.20.0
-    - pytorch >=1.8.1
+    - pytorch >=1.10.0
     - lightning-utilities >=0.8.0
```

```suggestion
    - packaging >17.1
    - lightning-utilities >=0.8.0
```
https://github.com/Lightning-AI/torchmetrics/blob/v1.4.0.post0/requirements/base.txt#L5

---

`packaging` isn't used outside their CI on github https://github.com/search?q=repo%3ALightning-AI%2Ftorchmetrics%20packaging&type=code